eng/Versions.props

```diff
@@ -58,6 +58,7 @@
     <MicrosoftAspNetCoreApp60Version>$(MicrosoftAspNetCoreAppRuntimewinx64Version)</MicrosoftAspNetCoreApp60Version>
   </PropertyGroup>
   <PropertyGroup Label="Manual">
+    <AzureIdentityVersion>1.6.0</AzureIdentityVersion>
```

Comment:
Just FYI, this might increase the payload significantly: https://www.nuget.org/packages/Azure.Identity/#dependencies-tab

Reply:
I compared published output between release/6.x vs my branch. This adds about 1.75MiB